packages/host/app/services/command-service.ts

```diff
@@ -0,0 +1,68 @@
+import Service from '@ember/service';
+
+type Command = 'patch';
+
+class CommandHandler {
+  handler: any;
```

Comment:
let's do a better job typing this. this is just a function right? (I think you wanted this as an async function?)

Reply:
No this ended up more bookkeeping than anything else, it's the object that's registering/unregistering the callbacks - that's why it ended up as any as this code is just using it as a reference so things can be cleaned up. I think there's a neater way.